<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>50</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>洛谷题目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>来做梦题目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>洛谷比赛</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>来做梦周赛</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>517题目</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>新建什么题目？</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_entry">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>99</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_luogucontest">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_luogucontext">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>洛谷比赛ID？</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_luogucontext">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_luogucontext_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;题目数？&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_luogucontext_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_gonoicontext">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_gonoicontext">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>周赛ID？</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_gonoicontext">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_gonoicontext_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>周赛组？</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>C（语法组）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>J（入门组）</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S（提高组）</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>爷要自己配置！</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_gonoiproblem">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_gonoiproblem">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;来做梦题号？&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_gonoiproblem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_luoguproblem">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_luoguproblem">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;洛谷题号？&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_luoguproblem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_517problem">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_517problem">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;517题号？&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_517problem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
